app/views/services/_flow_layout.html.erb

```diff
@@ -20,10 +20,10 @@
   <% else %>
 
     <% if item[:type] == 'flow.branch' %>
-      <article class="flow-item flow-branch" aria-label="<%= item[:title] %>" data-fb-id="<%= item[:uuid] %>">
-        <%= image_pack_tag "diamond.svg",  :usemap => "#thumbnail-#{item[:uuid]}" %>
+      <article class="flow-item flow-branch" data-fb-id="<%= item[:uuid] %>">
+        <%= image_pack_tag "diamond.svg",  usemap: "#thumbnail-#{item[:uuid]}", alt: "Branching diamond", role: "presentation" %>
         <map name="thumbnail-<%= item[:uuid] %>" aria-hidden="true">
-          <area shape="poly" coords="0,72.5 100,0 200,72.5 100,145" alt="" href="<%= edit_branch_path(service.service_id, item[:uuid]) %>">
+          <area shape="poly" coords="0,72.5 100,0 200,72.5 100,145" alt="Branching diamond" role="presentation" href="<%= edit_branch_path(service.service_id, item[:uuid]) %>" tabindex="-1">
```

Comment:
Since there's aria-hidden=true on this element, I'd have to look up to see if anything inside is even considered relevant/exposed to Accessible users??

I would expect the alt value to be same 'redundant' issue as the one specified above, however. 

My read aloud (guessing) expectation of this area of code (image+map) would be something like:
"blah... Branching diamond... blah... Branching diamond"

If that's what I'd get as a user experience then I'd file it under 'somewhat annoying' :-)